demo/fizz_buzz.py

```diff
@@ -0,0 +1,8 @@
+def fizz_buzz(n):
+    for i in range(1, n + 1):
+        if i % 3 == 0:
```

Comment:
「FizzBuzz」の条件が欠落しています。
15の倍数（3と5の両方の倍数）のときに「FizzBuzz」を出力する必要があります。

```suggestion
        if i % 15 == 0:
            print("FizzBuzz")
        elif i % 3 == 0:
```